src/components/CalendarMonthGrid.jsx

```diff
@@ -386,6 +400,13 @@ export default withStyles(({ reactDates: { color, zIndex } }) => ({
   CalendarMonthGrid__vertical_scrollable: {
     margin: '0 auto',
     overflowY: 'scroll',
+    ...noScrollBarOnVerticalScrollable && {
```

Comment:
could we add parens here to make it clear that the entire thing is spread, not just the first conditional?